docs/src/3.5.BrowserSupport.mdx

```diff
@@ -0,0 +1,10 @@
+# Browser Support
+
+Currently we support the following browsers/versions:
+
+- Chrome 71+
+- Firefox 64+
+- Desktop Safari 11+
+- Opera 57+
+
+Some unsupported browsers may require polyfill to work. We'll add more support in the future.
```

Comment:
Small language thing: "polyfill" -> either "a polyfill" or "polyfills" (probably the latter makes most sense?)